src/riak_core_coverage_fsm.erl

```diff
@@ -206,20 +212,27 @@ maybe_start_timeout_timer(Timeout) ->
     ok.
 
 %% @private
-find_plan(#vnode_coverage{}=Plan, _NVal, _PVC, _ReqId, _Service, _Request) ->
-    riak_core_coverage_plan:interpret_plan(Plan);
-find_plan({colocated, CMod}, NVal, PVC, ReqId, Service, Request) ->
-    CMod:create_plan(NVal,
-                     PVC,
-                     ReqId,
-                     Service,
-                     Request);
-find_plan(VNodeTarget, NVal, PVC, ReqId, Service, _Request) ->
-    riak_core_coverage_plan:create_plan(VNodeTarget,
-                                        NVal,
-                                        PVC,
-                                        ReqId,
-                                        Service).
+find_plan(_Mod, #vnode_coverage{}=Plan, _NVal, _PVC, _ReqId, _Service,
+          _Request) ->
+    interpret_plan(Plan);
+find_plan(Mod, Target, NVal, PVC, ReqId, Service, Request) ->
+    Mod:create_plan(Target, NVal, PVC, ReqId, Service, Request).
+
+%% @private
+%% Take a `vnode_coverage' record and interpret it as a mini coverage plan
+-spec interpret_plan(vnode_coverage()) ->
+                            {list({index(), node()}),
```

Comment:
Not something to hold back the PR, but I think we should define an alias for both `{index(), node()}` and `{index(), list(index())|tuple()}` and put them in a module (not in hrl file!).